src/components/StudyPane/Passage/StropheFunctions.tsx

```diff
@@ -0,0 +1,86 @@
+import { HebWord, PassageData} from "@/lib/data";
+
+export const newStropheAction = (wordArray:HebWord[], wordIdNumber:number):HebWord[] => {
+    for(let i = 0; i<wordArray.length; i++){
```

Comment:
Let's keep the spacing consistent in our code.

Please use this coding guideline:
> for (xxxx) {
> }